dateparser/data/date_translation_data/es.py

```diff
@@ -495,6 +495,12 @@
         },
         {
             "un": "1"
+        },
+        {
+            "(?:12\\s+)?mediodía": "12:00"
+        },
+        {
+            "(?:12\\s+)?medianoche": "00:00"
         }
```

Comment:
@christianfm10 please add a trailing comma here and also an extra newline at the end of the file. Thanks

Reply:
This file shouldn't be edited directly, instead he should run the `write_complete_data.py` script that generates the json file from the `.yaml` file.